### DISCUSSION TITLE
Make "seek to timecode" bring up a blank menu

### DISCUSSION THREAD

**Discussion by sunflowersim** [mifi/lossless-cut#1988] [Ideas]
Hi,
First wanted to say thanks for an amazing, god-tier program. 
Was wondering if you would consider making it an option to have the "seek to timecode" menu (upon hitting G) present with a blank field, instead of the current timecode. Having the current timecode is not particularly necessary for the seek function (at least, in my experience), and it means I have to ctrl+a + delete every time.
Apologies in advance if this is already an option and I missed it in my search of the settings, or it's been asked before and won't be implemented.

Thanks again!

**Comment by mifi** [comment]
hi. this is the first time i heard of this request, so i'm not sure it's a common one. but I will implement a change where it will auto-select the text after the dialog opens. that way you can just start typing as if it was empty.

**Reply by sunflowersim** [reply]
Of course, that's a more elegant option. Thanks a lot, I'm looking forward to the update :)
